### ISSUE TITLE
Cannot allocate memory - master branch

### ISSUE THREAD

**Issue by axeljerabek** [Motion-Project/motionplus#172]
### Did you read the guide?

Yes

### What is the base version number of MotionPlus being used?

0.2.x

### What was the install method?

Built from source code

### What is base architecture?

x86/x64

### What is the distro being used?

Ubuntu

### Disto version number

24.04

### Camera/Sound type(s) being used?

Network camera with RTSP, Network camera with HTTP

### Describe the issue/problem and steps to reproduce

After starting motionplus, the logfile says that it cannot allocate memory, see log output.
I have read your post about the instable master branch. But maybe my finds are helping you in debugging.

### Relevant MotionPlus log output (at log_level 8)

```shell
Jul 23 12:31:50 [NTC][ALL][motionplus] mytranslate_init: Language: English
Jul 23 12:31:50 [NTC][STR][motionplus] webu_start: Starting webcontrol on port 8081
Jul 23 12:31:50 [EMG][ALL][motionplus] mymalloc: Could not allocate 106147888165825 bytes of memory!: Cannot allocate memory
Jul 24 08:59:36 [NTC][ALL][motionplus] mytranslate_init: Language: English
Jul 24 08:59:36 [NTC][STR][motionplus] webu_start: Starting webcontrol on port 8081
Jul 24 08:59:36 [EMG][ALL][motionplus] mymalloc: Could not allocate 96003440616385 bytes of memory!: Cannot allocate memory
```

**Comment by Mr-Dave** [comment]
Can you provide the full log at the debug level?  Does it happen all the time or is it intermittent?

**Comment by Mr-Dave** [comment]
Thanks.  

This one has me stumped.  It looks like it is [this line](https://github.com/Motion-Project/motionplus/blob/0f86c4030f48c25039d8303d26b1d9a9ca41f260/src/webu.cpp#L391) but that calculation should only result in the number 240.  Somehow passing the result into the [mymalloc](https://github.com/Motion-Project/motionplus/blob/0f86c4030f48c25039d8303d26b1d9a9ca41f260/src/util.cpp#L183) function ends up with trillions on your computer....

**Comment by Mr-Dave** [comment]
Can you pull again to get the most recent commits to see if it resolves the issue?

**Comment by axeljerabek** [comment]
Just did, but still got the error:

Jul 25 06:59:27 [DBG][ALL][motionplus] motpls_ntc: v4l2   : available
Jul 25 06:59:27 [DBG][ALL][motionplus] motpls_ntc: webp   : available
Jul 25 06:59:27 [DBG][ALL][motionplus] motpls_ntc: libcam : available
Jul 25 06:59:27 [DBG][ALL][motionplus] motpls_ntc: mysql  : not available
Jul 25 06:59:27 [DBG][ALL][motionplus] motpls_ntc: MariaDB: available
Jul 25 06:59:27 [DBG][ALL][motionplus] motpls_ntc: sqlite3: not available
Jul 25 06:59:27 [DBG][ALL][motionplus] motpls_ntc: pgsql  : not available
Jul 25 06:59:27 [DBG][ALL][motionplus] motpls_ntc: nls    : available
Jul 25 06:59:27 [DBG][ALL][motionplus] motpls_ntc: alsa   : available
Jul 25 06:59:27 [DBG][ALL][motionplus] motpls_ntc: fftw3  : available
Jul 25 06:59:27 [NTC][STR][motionplus] webu_start: Starting webcontrol on port 8081
Jul 25 06:59:27 [DBG][ALL][motionplus] util_parms_add: webcontrol_actions:>event< >on<
Jul 25 06:59:27 [DBG][ALL][motionplus] util_parms_add: webcontrol_actions:>snapshot< >on<
Jul 25 06:59:27 [DBG][ALL][motionplus] util_parms_add: webcontrol_actions:>pause< >on<
Jul 25 06:59:27 [DBG][ALL][motionplus] util_parms_add: webcontrol_actions:>restart< >on<
Jul 25 06:59:27 [DBG][ALL][motionplus] util_parms_add: webcontrol_actions:>stop< >on<
Jul 25 06:59:27 [DBG][ALL][motionplus] util_parms_add: webcontrol_actions:>config_write< >on<
Jul 25 06:59:27 [DBG][ALL][motionplus] util_parms_add: webcontrol_actions:>camera_add< >on<
Jul 25 06:59:27 [DBG][ALL][motionplus] util_parms_add: webcontrol_actions:>camera_delete< >on<
Jul 25 06:59:27 [DBG][ALL][motionplus] util_parms_add: webcontrol_actions:>config< >on<
Jul 25 06:59:27 [DBG][ALL][motionplus] util_parms_add: webcontrol_actions:>ptz< >on<
Jul 25 06:59:27 [DBG][ALL][motionplus] util_parms_add: webcontrol_actions:>movies< >on<
Jul 25 06:59:27 [DBG][ALL][motionplus] util_parms_add: webcontrol_actions:>action_user< >on<
Jul 25 06:59:27 [EMG][ALL][motionplus] mymalloc: Could not allocate 98227385172929 bytes of memory!:

**Comment by Mr-Dave** [comment]
Found the bug.  Current master should resolve this.

**Comment by axeljerabek** [comment]
Yay! Thanks!

**Comment by Mr-Dave** [comment]
closed via f48a59f

**Comment by axeljerabek** [comment]
confirm
